services/scan/src/precinct_scanner_app.ts

```diff
@@ -99,8 +39,8 @@ export async function buildPrecinctScannerApp(
 ): Promise<Application> {
   const { store } = workspace;
 
-  // Try loading interpreter configuration from the store on startup
-  await updateInterpreterConfig(interpreter, workspace);
+  // Trigger the layouts to be cached in the store
+  await store.loadLayouts();
```

Comment:
Do we need to do this here? Doesn't it happen automatically when the store is initialized now?

Reply:
oops! meant to remove that.